datahub-web-react/src/app/shared/tags/TagGroup.tsx

```diff
@@ -0,0 +1,113 @@
+import { Modal, Tag } from 'antd';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { FetchResult } from '@apollo/client';
+
+import { useEntityRegistry } from '../../useEntityRegistry';
+import { EntityType, GlobalTags, GlobalTagsUpdate } from '../../../types.generated';
+import { UpdateDatasetMutation } from '../../../graphql/dataset.generated';
+import { convertTagsForUpdate } from './utils/convertTagsForUpdate';
+import AddTagModal from './AddTagModal';
+
+type Props = {
+    uneditableTags?: GlobalTags | null;
```

Comment:
Question: Would these props be better suited as an Array<Tag>? Why should this component know about the wrapping GlobalTags object... can be considered async to this PR

Reply:
Yeah, I tend to agree. Let's do that as a followup